cmd/estuary-shuttle/main.go

```diff
@@ -1184,11 +1184,16 @@ func (s *Shuttle) addrsForShuttle() []string {
 }
 
 func (s *Shuttle) createContent(ctx context.Context, u *User, root cid.Cid, fname string, cic util.ContentInCollection) (uint, error) {
+	bserv := blockservice.New(s.Node.Blockstore, s.Node.Bitswap)
```

Comment:
I think we do (this was a suggestion from @whyrusleeping IIRC) since this is ran on the shuttles, while `hadleCreateContent` runs on the main estuary node.